SiteOps/get_ff.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# get_ff.py - get the path and tape family for the incoming files in the dataset
+#
+# Usage:
+#
+#    get_ff.py data [data ...]
+#
+#    data could be a dataset name or a block name
+#
+#    the format of data is like the followings
+#
+#    dataset name is a "/" delimited path, the last term is the type
+#    block name is a dataset name with #<block>
+#
+import sys
+import os
+
+from rucio.client.didclient import DIDClient
```

Comment:
No need to import two different clients. Client will do it.

Reply:
use rucio.client.Client()